patches/picolibc.patch

```diff
@@ -1,11 +1,11 @@
 diff --git a/meson.build b/meson.build
-index 62384e63c..a242da5bf 100644
```

Comment:
I don't see any significant changes to this file so please remove it from the commit.